code/datums/components/mind_linker.dm

```diff
@@ -194,7 +194,7 @@
 	var/mob/living/linker_parent = linker.parent
 
 	var/message = sanitize(tgui_input_text(owner, "Enter a message to transmit.", "[linker.network_name] Telepathy"))
-	if(!message || QDELETED(src) || QDELETED(owner) || owner.stat == DEAD)
+	if(!message || QDELETED(src) || QDELETED(owner) || owner.stat >= UNCONSCIOUS)
```

Comment:
Change `IsAvailable` to check for `>= UNCONSCIOUS` too so we don't have the same issue here.

Reply:
i had to check for <= UNCONSCIOUS because the original variable was looking for != dead

I tested it and it works while conscious and doesnt work when unconscious.